README.md

```diff
@@ -1,2 +1,124 @@
+- **AD:** The AD folder contains all the templates specific to the AD consortium.
+- **PsychENCODE:** The PsychENCODE folder contains all the templates specific to the PsychENCODE consortium.
+
+## Annotation Schema
+
+Annotation schema can be found in the schema_annotations folder and are organized by consortium.
+
+- **AD:** The AD annotation schema is not currently set on any projects and will be updated to a new format.
+
+## Code
+
+There are python scripts in the code/python folder for generating metadata templates and annotation tables based on the metadata template schemas registered in Synapse.
+
+#### Annotations Table
+
+create_Syn_table_from_Syn_schemas.py will generate a Synapse table of all terms found in a set of metadata templates. The set is determined by consortium using the config file (config/schemas.yml). There are options to create a new table or overwrite an existing table.
+
+Parameters:
+
+- `--config_file <YAML file>`: Path to the config file.
+- `--consortium <consortium>`: The consortium to create the table for; will only gather terms in registered templates for that consortium.
+- `new_table`  OR `overwrite_table`: Choose to create a new table or overwrite an existing table.
+  - For `new_table`
+    - `--parent_synapse_id <synID>`: synID for project that table should be created in.
+    - `--synapse_table_name <table name>`: Name for the new table.
+  - For`overwrite_table`
+    - `--table_synapse_id <synID>`: synID of table to overwrite.
+
+#### Generate Metadata Template
+
+create_template_from_Syn_schema.py will generate either a .csv or .xlsx metadata template based on a registered metadata schema. Note that this script will not upload the file to Synapse automatically.
+
+Parameters:
+
+- `<registered schema id>`: id of the registered metadata schema, including organization (e.g. sysbio.metadataTemplates_assay.STARRSeq).
+- `<output file path>`: Full path to the where the template file should be output.
+- `csv`  OR `excel`: Choose output as .csv (csv) or .xlsx (excel). A csv file will only include a header with the metadata keys. An excel file will include three sheets: one with a header of metadata keys, one with a dictionary of key descriptions, and one with a dictionary of allowed values plus their descriptions.
+
+### Docker
+
+This repo contains a Dockerfile that can be used to build a docker image locally. Alternatively, the docker image is on Docker Hub under [aryllen/sysbioDCCjsonschemas](https://hub.docker.com/repository/docker/aryllen/sysbiodccjsonschemas).
+
+##### Build Image Locally
+
+If you'd like to build the docker image locally, clone this repo and open a terminal in the sysbioDCCjsonschemas folder. Then build image and run interactively.
+
+```bash
+git clone https://github.com/Sage-Bionetworks/sysbioDCCjsonschemas.git
+cd sysbioDCCjsonschemas
+docker build -t sysbiodccjsonschemas .
+docker run --rm -it sysbiodccjsonschemas
+```
+
+##### Pull Existing Image
```

Comment:
Are there situations where it would be better to pull an existing image vs build locally, or vice versa?

Reply:
The Docker Hub version may not be up to date with the repo. I haven't added a GH Action to auto-deploy this, which means I have to manually keep it up to date for now.

If you are wanting to use it to generate templates, then there's two situations where I might not have an up to date version ready for you in time:
- if you add a new key to an existing template
- add a new template altogether

For generating annotation tables, this would matter if you update the config with a new template.

Reply:
Cool, thank you! I might add that info to the readme in the so I don't have to bug you every 4 months when I forget what the difference is.